client/src/components/Flash/index.js

```diff
@@ -38,7 +44,7 @@ function Flash({ flashMessage, onClose }) {
             className='flash-message'
             onDismiss={handleClose}
           >
-            {needsTranslating ? t(`${message}`) : message}
+            {needsTranslating ? t(`${message}`, variables) : message}
```

Comment:
Do we need this logic any more?  If `t` returns the key when there's no corresponding entry then we can just use 
```suggestion
           t(message, variables)
```
where I've also removed the coercion of `message` to a string.  As far as I can see it already is one.

Reply:
Good call.